cpp/tests/io/csv_test.cpp

```diff
@@ -860,16 +882,16 @@ TEST_F(CsvReaderTest, SkipRowsXorSkipFooter)
 {
   std::string buffer = "1,2,3";
 
-  cudf_io::read_csv_args skiprows_args{cudf_io::source_info{buffer.c_str(), buffer.size()}};
-  skiprows_args.header = -1;
-  // only set skiprows
-  skiprows_args.skiprows = 0;
+  cudf_io::csv_reader_options skiprows_args =
+    cudf_io::csv_reader_options::builder(cudf_io::source_info{buffer.c_str(), buffer.size()})
+      .header(-1)
+      .skiprows(0);
```

Comment:
It's the default now, needs to be set to something else.
```suggestion
      .skiprows(1);
```